README.md

```diff
@@ -0,0 +1,26 @@
+This is a starter template for [Ionic](http://ionicframework.com/docs/) projects.
+
+## How to use this template
+
+*This template does not work on its own*. The shared files for each starter are found in the [ionic2-app-base repo](https://github.com/ionic-team/ionic2-app-base).
+
+To use this template, either create a new ionic project using the ionic node.js utility, or copy the files from this repository into the [Starter App Base](https://github.com/ionic-team/ionic2-app-base).
+
+### With the Ionic CLI:
+
+Take the name after `ionic2-starter-`, and that is the name of the template to be used when using the `ionic start` command below:
+
+```bash
+$ sudo npm install -g ionic cordova

```

Comment:
The commands work for me, for n00b like me I think this should be the important part and should be put to the front of the README.